### PR TITLE
Logging cleanups

### PR DESCRIPTION
**Pull request by aboodman**

### CODE REVIEW

File: backend/rds.ts [hunk 1]
```diff
@@ -154,7 +154,7 @@ async function executeStatementInDatabase(
       name,
       value,
     }));
-  console.log("Executing", database, sql, params);
+  console.log("Executing", database, sql, JSON.stringify(params, null, ''));
```
Comment (aboodman) [inline backend/rds.ts:157]: These were creating too much noise in the logs pretty-printed. There's probably some other way to do this but it works for now.

File: pages/api/replicache-push.ts [hunk 2]
```diff
@@ -99,16 +100,17 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
         }
 
         const next = push.mutations[i + 1];
-        console.log("Considering for collapse", mutation, next);
+        console.log("Considering for collapse", JSON.stringify(mutation, null, ''), JSON.stringify(next, null, ''));
```
Comment (arv) [inline pages/api/replicache-push.ts:103]: Just curious how you end up with both `"` and `'` on the same line? Have you not setup VSCode to format on save?

**Review by arv — COMMENTED** [review]
LGTM

File: pages/api/replicache-push.ts (hunk 2 above)
Comment (aboodman) [reply pages/api/replicache-push.ts:103]: No idea. What extensions do you use? I have prettier with the default settings.

Comment (aboodman) [reply pages/api/replicache-push.ts:103]: This line has been removed and there's no more single quotes in the file.

Comment (arv) [reply pages/api/replicache-push.ts:103]: `command` `,` -> `format on save` -> ✅

Comment (arv) [reply pages/api/replicache-push.ts:103]: I also have `esbenp.prettier-vscode`  installed if that matters.

**Pull request merged**